common.go

```diff
@@ -79,18 +79,9 @@ func NewIdentifier(value string) *IdentifierInput {
 	}
 }
 
-func NewEmptyString() *graphql.String {
-	s := graphql.String("")
-	return &s
-}
-
 func NewString(value string) *graphql.String {
-	var output *graphql.String = nil
```

Comment:
This refactor is a bit scary - but i called it out in the changelog as a possible BREAKING CHANGE.

We'll want to see all the places in Terraform where we use this (if any) and ensure they still work